device_services/generic/thermostat.md

```diff
@@ -24,6 +24,9 @@ Thermostat service is used to control the temperature in a room or a building an
 | in   | cmd.sensing.set         | string     |             |             | Sets the `sensing` type of the device, one of the valuesdeclared in `sup_sensing`.                                                                   |
 | in   | cmd.sensing.get_report  | null       |             |             | Requests the `sensing` type of the device.                                                                                                           |
 | out  | evt.sensing.report      | string     |             |             | Reports the `sensing` type of the device, one of the values declared in [`sup_sensing`](#service-properties) property.                               |
+| -    |                         |            |             |             |                                                                                                                                                      |
```

Comment:
This should most likely BE NOT part of thermostat service but a dedicated alarm service and have an alarm event similar to existing alarms.